strings/base_array.h

```diff
@@ -223,6 +237,11 @@ WINRT_EXPORT namespace winrt
     template <typename C, size_t N> array_view(std::array<C, N>& value) -> array_view<C>;
     template <typename C, size_t N> array_view(std::array<C, N> const& value) -> array_view<C const>;
 
+#ifdef __cpp_lib_span
+    template <typename C, std::size_t extent> array_view(std::span<C, extent>& value) -> array_view<C>;
```

Comment:
cppwinrt uses the undecorated integral types like `size_t` from `stdint.h` so no need for `std...`.